github/provider.go

```diff
@@ -145,7 +146,7 @@ func Provider() terraform.ResourceProvider {
 			"github_ip_ranges":                       dataSourceGithubIpRanges(),
 			"github_membership":                      dataSourceGithubMembership(),
 			"github_organization":                    dataSourceGithubOrganization(),
-			"github_actions_organization_secrets":    dataSourceGithubActionsOrganizationSecrets(),
+			"github_organization_ip_allow_list":      dataSourceGithubOrganizationIpAllowList(),
```

Comment:
sorted the resources by moving actions resource to top; then added new the ip allow list resource